src/lambda_function.py

```diff
@@ -88,6 +92,19 @@ def lambda_handler(event, context):
         object['results'].append({'type': 'polarity',     "outcome": {"error" : "The polarity score could not be calculated."}})
         object['results'].append({'type': 'objectivity', "outcome": {"error" : "The objectivity score could not be calculated."}})
 
+    logger.info(f'LambdaFunction: Trying to get bias score...')
+    if 'error' in credibilityresult['outcome']:
+        cred_input = -1
+    else:
+        cred_input = credibilityresult['outcome']['score']
+    try:
+        biasscoreresult = bs.getBiasScore(cred_input, sentanalysisresult['polarity'], sentanalysisresult['subjectivity'])
```

Comment:
@ANA-POTJE While there's no error handling for when polarity/subjectivity aren't set, I think the `try` will catch this exception in the meantime... not the most elegant but approving to get this code into production.

@GCJyo 
- [ ] Please maybe make a ticket to add better error handling to the function.

Reply:
yeah that's a good idea

Reply:
@jsjohnstone - I think the getBiasScore.py should return something if there is credibility but no polarity / subjectivity.
@GCJyo - Am I correct? 
Otherwise it will be an easy change on lambda function, but first need to know from you ...

Reply:
when Credibility = -1 , BiasScore = Polarity & Subjectivity
if there is 'no' Polarity or Subjectivity or both values ( no score available) then an error would be required - this is common for other scores too. we can show a message ' unable to progress / unavailable'.  can log as an issue in our tracker  
Zero - is valid on subjectivity, polarity and BiasScore will give a score

Reply:
@GCJyo  so I understand that there is no changes required in your code ...
If there is a valid credibility but no polarity or no subjectivity, no Bias Score should be calculated - correct?
If you confirm I am happy to block the call to your code in the lambda function, and create a new PR